src/content/concepts/index.md

```diff
@@ -45,7 +45,7 @@ For a better understanding of the ideas behind module bundlers and how they work
 
 ## Entry
 
-An __entry point__ indicates which module webpack should use to begin building out its internal [dependency graph](/concepts/dependency-graph/). webpack will figure out which other modules and libraries that entry point depends on (directly and indirectly).
+An __entry point__ indicates which module webpack should be used to begin building out its internal [dependency graph](/concepts/dependency-graph/). webpack will figure out which other modules and libraries that entry point depends on (directly and indirectly).
```

Comment:
This doesnt read right

Reply:
Yes, the current version is fine as it is. I misread "module webpack" as compound words earlier.

Reply:
no problem, glad i could help